src/masonite/drivers/session/RedisDriver.py

```diff
@@ -81,25 +86,25 @@ def save(
         for key in deleted_flashed:
             self.delete(f"{self.get_flash_prefix()}{key}")
 
-    def get_session_namespace(self, session_id=None):
+    def get_session_namespace(self, session_id=None) -> str:
         if not session_id:
             request = self.application.make("request")
             session_id = request.cookie("SESSID")
         namespace = self.options.get("namespace", "")
         namespace += ":" if namespace else ""
         return f"{namespace}session:{session_id}:"
 
-    def get_data_prefix(self):
+    def get_data_prefix(self) -> str:
         return f"{self.get_session_namespace()}data:"
 
-    def get_flash_prefix(self):
+    def get_flash_prefix(self) -> str:
         return f"{self.get_session_namespace()}flash:"
 
     def helper(self) -> "RedisDriver":
         """Use to create builtin helper function."""
         return self
 
-    def get(self, key, default=None, **options):
+    def get(self, key, default=None, **options) -> str:
```

Comment:
```suggestion
    def get(self, key: str, default: Any = None, **options) -> str:
```

Reply:
For set() you can type hint like this:
```python
def set(self, key: str, value: Any, time: int = None):
```

For has() you can type hint like this:
```python
def has(self, key: str) -> bool:
```

Reply:
Yeah I see ... For some reason I thought "type hinting" meant the hint for the return value of methods.
I learnt something new today... 
Cheers

Done